readme_pages/algorithms.md

```diff
@@ -0,0 +1,85 @@
+# List of Supported and Planned Algorithms
+
+## Supported Graph
+
+| Type       | Description                                                 |
+| ---------- | ----------------------------------------------------------- |
+| Graph      | A directed or undirected Graph (use directed={True, False}) |
+| Multigraph | A Graph with multiple edges between a vertex pair           |
+|            |                                                             |
+
+ALL Algorithms support Graphs and MultiGraph (directed and undirected)
+
+---
+
+<br>
+
+# Supported Algorithms
+
+_Italic_ algorithms are planned for future releases.
+
+Note: Multi-GPU, or MG, includes support for Multi-Node Multi-GPU (also called MNMG).
+
+| Category          | Algorithm                          | Scale               | Notes                                                           |
+| ----------------- | ---------------------------------- | ------------------- | --------------------------------------------------------------- |
+| Centrality        |                                    |                     |                                                                 |
+|                   | Katz                               | __Multi-GPU__ |                                                                 |
+|                   | Betweenness Centrality             | **Multi-GPU** | MG as of 22.12                                                  |
```

Comment:
This did not make it into 22.12 - should be marked Single-GPU and the note be MG planned for 23.02

Reply:
fixed